.github/scripts/requirements.txt

```diff
@@ -0,0 +1,14 @@
+## @file
+# GitHub Helpers Python PIP requirements file
+#
+# This file provides the list of python components used in GitHub scripts in this repository.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+edk2-pytool-library==0.*
+edk2-pytool-extensions==0.*

```

Comment:
Where are edk2-pytool modules are used?

Reply:
Only for the `RunPythonScript()` call. I can drop edk2-pytool-extensions. If you'd really prefer I call `GetMaintainer.py` without pytools, I could do that as well. With PIP module caching, I don't think the pytool-library dependency will add much time though.

Reply:
Just want the dependencies to be accurate.

Reply:
I removed edk2-pytool-extensions.